src/plugins/discover/public/components/discover_grid/constants.ts

```diff
@@ -11,10 +11,10 @@ import { EuiDataGridStyle } from '@elastic/eui';
 // data types
 export const kibanaJSON = 'kibana-json';
 export const GRID_STYLE = {
-  border: 'all',
+  border: 'horizontal',
   fontSize: 's',
-  cellPadding: 's',
-  rowHover: 'none',
+  cellPadding: 'm',
+  rowHover: 'highlight',
```

Comment:
Similarly, we may want to wait to enable row highlighting until we alter the contents of the "Document" column to no longer use the EUI description list component styling. I say this because field names within the "Document" column to appear in a gray badge that is very difficult to see when combined with the new highlight (but will be addressed when we switch to bolding the field names and prepending them with field type tokens). Thoughts?

Reply:
I agree. @kertal do we have an issue tracking that? I couldn't find one and [#164287](https://github.com/elastic/kibana/issues/164287) doesn't cover it, maybe we should update it to include:

- [ ] Remove grey background from **EuiDescriptionList**'s `description` in table and make it bolder.

Reply:
working on an issue for that

Reply:
@MichaelMarcialis here's the issue #164634